plugins/3rd-party-optimizer/src/vite-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="vite/client" />
+
+declare module "virtual:yield-gtm-calls" {
+    const content: string
+    export default content
+}
```

Comment:
I'd say this belongs in an `index.d.ts` or `globals.d.ts` or something like that 🤔 This is for Vite only, and should be exactly this:

```ts
/// <reference types="vite/client" />

interface ViteTypeOptions {
    strictImportMetaEnv: unknown
}
```

Reply:
Update, I don't, it has to be vite-env. Otherwise:

```
6:26:17 PM [vite] Internal server error: Failed to parse source for import analysis because the content contains invalid JS syntax. If you are using JSX, make sure to name the file with the .jsx or .tsx extension.
```